packages/instrumenter/test/unit/mutators/method-expression-mutator.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { expect } from 'chai';
+
+import { methodExpressionsMutator as sut } from '../../../src/mutators/method-expression-mutator';
+import { expectJSMutation } from '../../helpers/expect-mutation';
+
+describe(sut.name, () => {
+  it('should have name "MethodExpression"', () => {
+    expect(sut.name).eq('MethodExpression');
+  });
+
+  it('should mutate forEach and map', () => {
+    expectJSMutation(sut, 'a.forEach()', 'a.map()');
```

Comment:
Mutating a "forEach" into a "map" would in my opinion almost always survive the test, or do you see a case where it would not ?